modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2835,6 +2853,25 @@
 	end
 
 
+	function m.externalWarningLevel(cfg)
+		if _ACTION >= "vs2022" then
```

Comment:
So probably dedicated file for vs2022 stuff.

Reply:
I combined the functions, moving this into a vcxproj file specifically for 2022 would be a different pattern than all the other vs-version specific stuff in this file which checks _ACTION.